workflow-templates/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Link Markdown
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  check_links:
+    name: Links
+    runs-on: ubuntu-latest
+    steps:
+      - name: Actions Checkout
+        uses: actions/checkout@v2
+      - name: Check Links
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
+        with: 
+          file-extension: '.md'
+          check-modified-files-only: 'yes'
```

Comment:
Just a note. I wouldn't use `yes` here because deletion of a `md` file can make other docs invalid despite they were not modified.

Reply:
"file extension" as ".md" is only one scenario, since it's possible that there are other extensions that are equally valid. Could you please instead use `categories` as defined here in [creating a workflow template](https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization#creating-a-workflow-template) and set the category to Markdown? Categories supported are listed here, [on line 3500 for markdown](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml).

I would also agree that for **link check** it shouldn't be modified files only, as there are other links that could break due to deletion. So take that one out.

Reply:
"file extension" as ".md" is only one scenario, since it's possible that there are other extensions that are equally valid. Could you please instead use `categories` as defined here in [creating a workflow template](https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization#creating-a-workflow-template) and set the category to Markdown? Categories supported are listed here, [on line 3500 for markdown](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml).

I would also agree that for **link check** it shouldn't be modified files only, as there are other links that could break due to deletion. So take that one out.

Reply:
Ok perfect, Im gonna set the extensions for the markdown:
extensions:
  - ".md"
  - ".markdown"
  - ".mdown"
  - ".mdwn"
  - ".mdx"
  - ".mkd"
  - ".mkdn"
  - ".mkdown"
  - ".ronn"
  - ".scd"
  - ".workbook"

Reply:
I set this in the ci.properties.json file